java/bench/spark/pom.xml

```diff
@@ -125,7 +125,7 @@
     <dependency>
       <groupId>org.objenesis</groupId>
       <artifactId>objenesis</artifactId>
-      <version>3.2</version>
+      <version>3.4</version>
```

Comment:
According to #830, `3.3`?

https://github.com/apache/spark/blob/45befc07d2a064ab2a279a113489ed5c66f7a69d/dev/deps/spark-deps-hadoop-3-hive-2.3#L208